### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by errbufferoverfl** [errbufferoverfl/garden.errbufferoverfl.me#3]
- [ ] Colour and contrast of broken links, does not meet WCAG standards (currently at 2.46).
- [ ] Colour and contrast of "See x more links", does not meet WCAG standards (currently at 3.05).
- [ ] Currently reading image is missing alt text.
- [ ] Currently reading link is missing a text label.
- [ ] The check boxes on the index are missing text labels.
- [ ] The Interactive Graph is not focus-able and does not have interactive semantics.
- [ ] The "Support Me on Ko-Fi iFrame is missing a text label.
